shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-fixture/src/test/java/org/apache/shardingsphere/test/integration/cases/assertion/IntegrationTestCaseAssertion.java

```diff
@@ -65,9 +66,26 @@
         Collection<SQLValue> result = new LinkedList<>();
         int count = 0;
         for (String each : Splitter.on(",").trimResults().splitToList(parameters)) {
-            List<String> parameterPair = Splitter.on(":").trimResults().splitToList(each);
+            List<String> parameterPair = parseParameter(each);
             result.add(new SQLValue(parameterPair.get(0), parameterPair.get(1), ++count));
         }
         return result;
     }
+    
+    private List<String> parseParameter(final String each) {
```

Comment:
The parameter `each` is hard to understand.

Reply:
I will rename it.